lib/mbsy/version.rb

```diff
@@ -1,3 +1,3 @@
 module Mbsy
-  VERSION = '2.0.0'
+  VERSION = '2.0.1'
```

Comment:
@brandoshmando this should be 2.1.0 if we're following semver since it's a backwards-compatible enhancement.